utils/contributors.js

```diff
@@ -59,6 +59,32 @@ const CONTRIBUTERS = [
         `,
         github: "https://github.com/debarati-06",
     },
+    {
+        handle_name: 'bsp2569',
+        name: "Bhabani Shankar Panigrahi",
+        image: "Passport size photo.jpeg",
```

Comment:
Image path is still incorrect. Also use a image name without spaces
Path should be `/images/file_name.jpg`